WordPress/WordPressAllTimeWidget/Tracks+AllTimeWidget.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+
+/// This extension implements helper tracking methods, meant for Today Widget Usage.
+///
+extension Tracks {
+    // MARK: - Public Methods
+    public func trackExtensionAccessed() {
+        trackExtensionEvent(.Accessed)
+    }
+
+    public func trackExtensionStatsLaunched(_ siteID: Int) {
+        let properties = ["site_id": siteID]
+        trackExtensionEvent(.StatsLaunched, properties: properties as [String: AnyObject]?)
+    }
+
+    public func trackExtensionConfigureLaunched() {
+        trackExtensionEvent(.ConfigureLaunched)
+    }
+
+    public func trackDisplayModeChanged(properties: [String: Bool]) {
+        trackExtensionEvent(.DisplayModeChanged, properties: properties as [String: AnyObject])
+    }
+
+    // MARK: - Private Helpers
+    fileprivate func trackExtensionEvent(_ event: ExtensionEvents, properties: [String: AnyObject]? = nil) {
+        track(event.rawValue, properties: properties)
+    }
+
+
+    // MARK: - Private Enums
+    fileprivate enum ExtensionEvents: String {
+        case Accessed           = "wpios_all_time_extension_accessed"
```

Comment:
Ah, you caught me copying & pasting. 😄 I can do that. If you don't mind, since I've branched twice since this PR, I'll do it in the one I'm currently working on.